examples/meta/src/distribution/histogram.sg

```diff
@@ -0,0 +1,18 @@
+File f_feats_train = csv_file("../../data/fm_train_dna.dat")
+
+ #![create_features]
+Features char_feats = string_features(f_feats_train, enum EAlphabet.DNA)
+Features embedded_feats = string_features(char_feats, 2, 3, 0, False)
+#![create_features]
+
+#![create_distribution]
+Distribution histogram = distribution("Histogram")
+histogram.set_features(embedded_feats)
+#![create_distribution]
+
+#![train_and_extract_values]
+histogram.train()
+RealVector histogram_array = histogram.get_real_vector("histogram")
```

Comment:
This histogram implementation calculates frequencies in log space, but it's not normalized, which is a bit weird?

Reply:
nvm, it was intentional, so maybe not
```C++
/** @brief Class Histogram computes a histogram over all 16bit unsigned
 * integers in the features.
 *
 * Values in histogram are absolute counts (logarithmic)
 */
```

Reply:
i guess this is as to avoid overflows?
But that was written well before my (or anyone else's apart from @sonney2k ) time